packages/apputils/src/commandlinker.ts

```diff
@@ -97,10 +79,8 @@ export class CommandLinker implements IDisposable {
    * using the `populateVNodeDataset` method in order to disconnect them from
    * executing their command/argument pair.
    */
-  disconnectNode(node: HTMLElement): HTMLElement {
-    node.removeAttribute(`data-${COMMAND_ATTR}`);
-    node.removeAttribute(`data-${ARGS_ATTR}`);
-    return node;
+  disconnectNode(node: HTMLElement): void {
+    AttachedProperty.clearData(node);
```

Comment:
We don't want to clear *all* property data, just the linkedCommandProperty data.

Reply:
I agree, but I don't see any `clear` function on `AttachedProperty`. Maybe a `WeakMap` would be the better choice.